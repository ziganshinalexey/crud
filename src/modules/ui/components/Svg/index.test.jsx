import React from 'react';
import {SvgIcon} from '.';
import {shallow} from 'enzyme';

describe('<Svg />', () => {
    it('should render', () => {
        const element = shallow(<SvgIcon src={{id: 'icon-name'}} />);
        expect(element.debug()).toMatchSnapshot();
    });
});
