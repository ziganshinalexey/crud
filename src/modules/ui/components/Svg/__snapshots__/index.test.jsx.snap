// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Svg /> should render 1`] = `
"<svg height=\\"1em\\" width=\\"1em\\">
  <use xlinkHref=\\"#icon-name\\" />
</svg>"
`;
